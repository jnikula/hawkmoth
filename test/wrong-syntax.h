/** C++ class inside a '.h'. */
class foo {
};
