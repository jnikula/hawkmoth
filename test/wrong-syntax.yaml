domain: c
directive-arguments:
  - wrong-syntax.h
example-title: C++ syntax in C source
example-priority: 1
