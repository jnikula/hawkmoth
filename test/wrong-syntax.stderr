ERROR: wrong-syntax.h:2: unknown type name 'class'
ERROR: wrong-syntax.h:2: expected ';' after top level declarator
